### ISSUE TITLE
Extract XAI methods from State of the Art on Validation Techniques for ML

### ISSUE THREAD

**Issue by cwmeijer** [dianna-ai/dianna-exploration#19]
Sub issue from #8.
Create an initial list of XAI methods to chose from. Extract XAI methods from the document below:

**Input**
Get references from: 

- Recent deliverable on "[State of the Art on Validation Techniques for ML](https://nlesc-my.sharepoint.com/:b:/g/personal/e_ranguelova_esciencecenter_nl/Ebg6opGveSVHsBahOSfD_J8BvIIIjVtZckhIPi8vWW5Bcg?e=S3BQek)", especially Tables 1 and 2 on pages 27 and 28

> 
> Old version can be found here:
> - My preliminary work overview summarizing some [OSS libraries and what methods they explain](https://github.com/NLeSC/XAI/blob/master/State-of-the-art/AI_Explainability_OSTools.docx)

**Output**
Add to [document](https://nlesc-my.sharepoint.com/:w:/g/personal/e_ranguelova_esciencecenter_nl/EctKDvZSDGlJqEfVaW6d-sgB3S8Yyou3oYcJMsF0wdrQlw?e=UYhR4X) to help to decide on the initial list.

**Comment by loostrum** [comment]
| Method | Comment |
| --------|---------- |
| ~~G-REX~~ | only accepts tabular data |
| ~~QII~~ | only accepts tabular data |
| ~~ELI5~~ | Not a method but a set of implementations? |
| LIME | Already in method list, added links to implementations |
| ~~Anchors~~ |only accepts tabular/text data |
| Real-time image saliency | Added notes to existing SA item|
| ASTRID | Added |
| KernelSHAP | part of existing SHAP |
| ~~TreeSHAP~~ | only accepts tabular data |
| ~~DeepLIFT~~ | DeepSHAP is enhanced version of DeepLIFT|
| DeepSHAP | Already in method list |
| Grad-Cam | Already in method list, added links to implementations |
| Deep visualization | Added |
